releng/tools/org.openjdk.jmc.util.releasenotes/src/org/openjdk/jmc/utils/releasenotes/Transform.java

```diff
@@ -32,15 +32,90 @@
  */
 package org.openjdk.jmc.utils.releasenotes;
 
-import com.sun.org.apache.xalan.internal.xslt.Process;
+import static java.nio.file.Files.*;
+import static java.nio.file.Files.newInputStream;
+import static java.nio.file.Files.newOutputStream;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayDeque;
+import java.util.Arrays;
+
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
 
 /**
  * Wrapper for Xalan to do an XSLT. Used to generate the release notes HTML.
  * <p>
  * Example: Transform -IN notes.xml -XSL stylesheet.xsl -OUT new_and_noteworthy.html
  */
 public class Transform {
+
 	public static void main(String[] args) {
-		Process._main(args);
+		final ArrayDeque<String> deque = new ArrayDeque<>(Arrays.asList(args));
```

Comment:
Style: Deque<String> deque = ...

Reply:
Changed, the idea was to use less imports as it is only used within this method. Would be an excellent `var` case..